internal/commands/app/init.go

```diff
@@ -21,6 +25,7 @@ func (cmd *CommandInit) Flags(fs *pflag.FlagSet) {
 	fs.StringVarP(&cmd.inputs.Name, flagName, flagNameShort, "", flagNameUsage)
 	fs.VarP(&cmd.inputs.DeploymentModel, flagDeploymentModel, flagDeploymentModelShort, flagDeploymentModelUsage)
 	fs.VarP(&cmd.inputs.Location, flagLocation, flagLocationShort, flagLocationUsage)
+	fs.Var(&cmd.inputs.ConfigVersion, flagConfigVersion, flagConfigVersionUsage)
```

Comment:
if we're going to add this, let's keep it consistent b/w `app init` and `app create` and also figure out how to make these "hidden" flags (cobra supports some notion of that, i believe)